Typescript builds failing

### What happened?

## Steps to Reproduce

Use  0.40.0 in a Typescript 5.1.3

└─ @opentelemetry/exporter-trace-otlp-proto@npm:0.40.0 [f7ec5]
   └─ @opentelemetry/otlp-proto-exporter-base@npm:0.40.0 [cb63c] (via npm:0.40.0 [cb63c])


## Expected Result

A working build

## Actual Result

node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:6:33 - error TS2503: Cannot find namespace 'protobuf'.

6     encode(message: T, writer?: protobuf.Writer): protobuf.Writer;
                                  ~~~~~~~~

node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:6:51 - error TS2503: Cannot find namespace 'protobuf'.

6     encode(message: T, writer?: protobuf.Writer): protobuf.Writer;
                                                    ~~~~~~~~

node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:7:20 - error TS2503: Cannot find namespace 'protobuf'.

7     decode(reader: protobuf.Reader | Uint8Array, length?: number): R;
                     ~~~~~~~~

## Additional Details


### OpenTelemetry Setup Code

_No response_

### package.json

```JSON
{
  "name": "@openapi-typescript-infra/service",
  "version": "0.0.0",
  "description": "An opinionated framework for building configuration driven services - web, api, or job. Uses OpenAPI, pino logging, express, confit, Typescript and Jest.",
  "main": "build/index.js",
  "scripts": {
    "test": "jest",
    "lint": "eslint .",
    "build": "tsc -p tsconfig.build.json && yarn dlx glob-chmod 755 build/bin/*",
    "watch": "tsc -p tsconfig.json -w --preserveWatchOutput",
    "clean": "npx rimraf ./build",
    "prepublishOnly": "yarn build",
    "prepack": "pinst --disable",
    "postpack": "pinst --enable",
    "postinstall": "husky install && coconfig"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/openapi-typescript-infra/service.git"
  },
  "bin": {
    "start-service": "./build/bin/start-service.js"
  },
  "config": {
    "coconfig": "@openapi-typescript-infra/coconfig"
  },
  "engines": {
    "node": ">=18"
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": "yarn eslint --cache --fix"
  },
  "keywords": [
    "service",
    "openapi",
    "express",
    "confit",
    "babel",
    "typescript",
    "jest"
  ],
  "author": "developers@pyralis.com>",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/openapi-typescript-infra/service/issues"
  },
  "release": {
    "branches": [
      "main"
    ]
  },
  "homepage": "https://github.com/openapi-typescript-infra/service#readme",
  "dependencies": {
    "@gasbuddy/confit": "^3.0.0",
    "@godaddy/terminus": "^4.12.0",
    "@opentelemetry/api": "^1.4.1",
    "@opentelemetry/api-metrics": "^0.33.0",
    "@opentelemetry/exporter-prometheus": "^0.40.0",
    "@opentelemetry/exporter-trace-otlp-proto": "^0.40.0",
    "@opentelemetry/instrumentation": "^0.40.0",
    "@opentelemetry/instrumentation-aws-sdk": "^0.34.3",
    "@opentelemetry/instrumentation-dns": "^0.31.5",
    "@opentelemetry/instrumentation-express": "^0.32.4",
    "@opentelemetry/instrumentation-generic-pool": "^0.31.4",
    "@opentelemetry/instrumentation-graphql": "^0.34.3",
    "@opentelemetry/instrumentation-http": "^0.40.0",
    "@opentelemetry/instrumentation-ioredis": "^0.34.3",
    "@opentelemetry/instrumentation-net": "^0.31.4",
    "@opentelemetry/instrumentation-pg": "^0.35.3",
    "@opentelemetry/instrumentation-pino": "^0.33.4",
    "@opentelemetry/sdk-metrics": "^1.14.0",
    "@opentelemetry/sdk-node": "^0.40.0",
    "@opentelemetry/semantic-conventions": "^1.14.0",
    "cookie-parser": "^1.4.6",
    "dotenv": "^16.3.0",
    "eventsource": "^1.1.2",
    "express": "next",
    "express-openapi-validator": "^5.0.4",
    "glob": "^8.1.0",
    "lodash": "^4.17.21",
    "minimist": "^1.2.8",
    "pino": "^8.14.1",
    "read-pkg-up": "^7.0.1",
    "rest-api-support": "^1.16.3",
    "shortstop-dns": "^1.1.0",
    "shortstop-handlers": "^1.1.1",
    "shortstop-yaml": "^1.0.0"
  },
  "devDependencies": {
    "@commitlint/cli": "^17.6.5",
    "@commitlint/config-conventional": "^17.6.5",
    "@openapi-typescript-infra/coconfig": "^3.0.2",
    "@types/cookie-parser": "^1.4.3",
    "@types/eventsource": "1.1.11",
    "@types/express": "^4.17.17",
    "@types/glob": "^8.1.0",
    "@types/jest": "^29.5.2",
    "@types/lodash": "^4.14.195",
    "@types/minimist": "^1.2.2",
    "@types/node": "^18.16.18",
    "@types/supertest": "^2.0.12",
    "coconfig": "^0.12.0",
    "eslint": "^8.43.0",
    "eslint-config-gasbuddy": "^7.2.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-jest": "^27.2.1",
    "husky": "^8.0.3",
    "jest": "^29.5.0",
    "lint-staged": "^13.2.2",
    "pino-pretty": "^10.0.0",
    "pinst": "^3.0.0",
    "supertest": "^6.3.3",
    "ts-jest": "^29.1.0",
    "ts-node": "^10.9.1",
    "typescript": "^5.1.3"
  },
  "packageManager": "yarn@3.2.3"
}
```


### Relevant log output

```shell
node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:6:33 - error TS2503: Cannot find namespace 'protobuf'.

6     encode(message: T, writer?: protobuf.Writer): protobuf.Writer;
                                  ~~~~~~~~

node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:6:51 - error TS2503: Cannot find namespace 'protobuf'.

6     encode(message: T, writer?: protobuf.Writer): protobuf.Writer;
                                                    ~~~~~~~~

node_modules/@opentelemetry/otlp-proto-exporter-base/build/src/platform/util.d.ts:7:20 - error TS2503: Cannot find namespace 'protobuf'.

7     decode(reader: protobuf.Reader | Uint8Array, length?: number): R;
                     ~~~~~~~~
```

--- a/experimental/CHANGELOG.md
+++ b/experimental/CHANGELOG.md
@@ -16,6 +16,7 @@ All notable changes to experimental packages in this project will be documented 
 ### :bug: (Bug Fix)
 
 * fix(exporter-logs-otlp-http): set useHex to true [#3875](https://github.com/open-telemetry/opentelemetry-js/pull/3875) @Nico385412
+  fix(otlp-proto-exporter-base): add missing type import [#3937](https://github.com/open-telemetry/opentelemetry-js/pull/3937) @pichlermarc
 
 ### :books: (Refine Doc)
 

--- a/experimental/packages/otlp-proto-exporter-base/src/platform/util.ts
+++ b/experimental/packages/otlp-proto-exporter-base/src/platform/util.ts
@@ -16,6 +16,7 @@
 
 import * as root from '../generated/root';
 import { ServiceClientType } from './types';
+import type * as protobuf from 'protobufjs';
 
 export interface ExportRequestType<T, R = T & { toJSON: () => unknown }> {
   create(properties?: T): R;

